.github/workflows/check_markdown.yml

```diff
@@ -1,7 +1,7 @@
 name: Check markdown
 
 on:
-    push:
+    pull_request:
         branches:
             - master
```

Comment:
If you look at other workflows, there should be working format of this

Reply:
Also, in `rector_ci.yaml`, there is a snippet that already handles autocommits.

On the other hand, maybe it's not needed to autocommit on everydocs change, as it might be annoying to contributors to always pull or force-push on bot commits.

What do you think?

Reply:
Do you think I need to copy/modify step at rector_ci.yaml ? https://github.com/rectorphp/rector/blob/6508aedd1321f5905c1d2728c9e1579071fd885a/.github/workflows/rector_ci.yaml#L45-L80 

The check-markdown command will keep the file if already follow coding standard, so it only commit when not follow coding standard.

Reply:
> so it only commit when not follow coding standard.

That's it the issue. If someone wants to contribute a new rule, they'd have to handle this fix manually or pull the commits commits from a bot. That is a big surprise and requires extra git skill.

I wonder, maybe it could be part of `composer.json` scripts and not CI.

```json
{
    "scripts": {
        "check-cs": "vendor/bin/ecs check --ansi",
        "fix-cs": [
            "vendor/bin/ecs check --fix --ansi",
            "bin/clear_posts_trailing_whitespaces.sh"
        ],
    }
}
```

Reply:
We had same troubles with Generated Documentation. It's not neccesary on every contributions and only bothers new contributors who just want to add that single rule.

So we decided to put that workflow into inactive and handle this manaully from time to time. It's not perfect, but developer experience has higher priority.

Reply:
ok, I will try add to composer script

Reply:
Thanks